rov/sensors/imu/Adafruit_BNO055/__init__.py

```diff
@@ -1 +0,0 @@
-from imu import IMU
```

Comment:
Move this stuff out of the folder, since we don't need the folder to hide the Adafruit library away anymore